pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -182,7 +183,7 @@ func (this *SnapshotManager) DeleteSnapshot(peID astrolabe.ProtectedEntityID) er
 	this.Infof("Step 1: Deleting the local snapshot, %s", peID.String())
 	err := this.DeleteProtectedEntitySnapshot(peID, false)
 	if err != nil {
-		this.Errorf("Failed to delete the local snapshot")
+		this.Errorf("Failed to delete the local snapshot: %v", err)
```

Comment:
What PEID/snapshot ID were we unable to delete?

Reply:
Add peid in the error message.